overrides/default.nix

```
@@ -855,6 +855,19 @@ lib.composeManyExtensions [
         }
       );
 
+      hidapi = super.hidapi.overridePythonAttrs (
+        old: {
+          buildInputs = old.nativeBuildInputs or [ ] ++ [
```

Shouldn't this be `nativeBuildInputs`?

```suggestion
          nativeBuildInputs = old.nativeBuildInputs or [ ] ++ [
```

---

Sorry for that mistake, but no, I think it should be buildInputs like so
```suggestion
          buildInputs = old.buildInputs or [ ] ++ [
```
Otherwise there is a linking error, libusb1 will be used at run time. 
I see that nixpkgs actually has it as propagatedBuildInputs, Is it better to use that?

---

Right. I think it should belong to `propagatedBuildInputs` afterall.